### PR TITLE
refactor: change jobManagement to new jobDefentions (MAPCO-5091)

### PR DESCRIPTION
**Pull request by razbroc**
| Question                | Answer                                                                          |
| ---------------- | -------------------------------------------------------------------------- |
| Bug fix         | ✖                                                                        |
| New feature     | ✖                                                                        |
| Breaking change | ✔                                                                        |
| Deprecations    | ✖                                                                        |
| Documentation   | ✖                                                                        |
| Tests added     | ✖                                                                        |
| Chore            | ✔                                                                       |

Related issues: MAPCO-5091

### CODE REVIEW

File: config/custom-environment-variables.json [hunk 1]
```diff
@@ -33,6 +33,21 @@
         "limit": "REQUEST_PAYLOAD_LIMIT"
       }
     },
+    "jobDefinitions": {
```
Comment (CL-SHLOMIKONCHA) [inline config/custom-environment-variables.json]: move outside the `server` scope

File: helm/values.yaml
Comment (CL-SHLOMIKONCHA) [inline helm/values.yaml:1]: - add `jobDefinitions: {}` to global scope.
- add `domain: ' '` to global scope.

**Review by CL-SHLOMIKONCHA — CHANGES_REQUESTED** [review]
You didnt change the `default.json` config file..

File: helm/values.yaml
Comment (razbroc) [reply helm/values.yaml:1]: ok

File: config/custom-environment-variables.json (hunk 1 above)
Comment (razbroc) [reply config/custom-environment-variables.json]: ok

**Review by CL-SHLOMIKONCHA — APPROVED** [review]

**Review by RonitKissis — APPROVED** [review]

**Pull request merged**